File: Lecture03/ListViews/app/src/main/java/com/codingblocks/listviews/MainActivity.java
package com.codingblocks.listviews;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

public class MainActivity extends AppCompatActivity {

    String[] courses = {
            "Android",
            "Webdev",
            "Launchpad",
            "Crux",
            "Gamedev",
            "Machine Learning",
            "Competitive Coding",
            "Django",
            "Angular JS",
            "React",
            "Android",
            "Webdev",
            "Launchpad",
            "Crux",
            "Gamedev",
            "Machine Learning",
            "Competitive Coding",
            "Django",
            "Angular JS",
            "React"
    };
    ListView lvCourses;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        lvCourses = findViewById(R.id.lvCourses);

//        ArrayAdapter<String> courseAdapter = new ArrayAdapter<>(
//                this,
//                android.R.layout.simple_list_item_1,
//                android.R.id.text1,
//                courses
//        );
        ArrayAdapter<String> courseAdapter = new ArrayAdapter<>(
                this,
                R.layout.list_item_course,
                R.id.tvCourseName,
                courses
        );

        lvCourses.setAdapter(courseAdapter);

    }
}

File: Lecture20/AnimationTransition/app/src/main/java/com/codingblocks/animationtransition/MainActivity.kt
package com.codingblocks.animationtransition

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var btn = MyButton(this)
        btn.text = "A"
        llBackground.addView(btn)

        llBackground.addView(MyButton(this).apply {
            text = "B"
        })

    }
}
